src/server.cpp

```diff
@@ -294,6 +301,12 @@ void Server::loadServerConfig()
     dice_value = config.value("value_type", "100").toInt();
     max_dice = config.value("max_dice","100").toInt();
     config.endGroup();
+
+    //Load discord webhook
+    config.beginGroup("Discord");
+    webhook_enabled = config.value("webhook_enabled", "false").toBool();
+    webhook_url = config.value("webhook_url", "Your webhook url here.").toString();
+    webhook_sendfile = config.value("webhook_sendfile", false).toBool();
```

Comment:
```suggestion
    webhook_sendfile = config.value("webhook_sendfile", false).toBool();
    config.endGroup();
```

Though not ending the config group at the last part may not cause too many problems.